modules/rosa-create-objects.adoc

```diff
@@ -500,6 +503,11 @@ $ rosa create machinepool --cluster=<cluster_name> | <cluster_id> --replicas=<nu
 |===
 |Option |Definition
 
+// Not yet supported but part of OSDOCS-7730
```

Comment:
@bchandra-ocp This is the part you're concerned about for XCMSTRAT-41 - there are additional improvements I can make to the Nodes book, but this is the main part.

Reply:
And just confirmed with Guilherme that this is ready, so it's uncommented now. :)